### PR TITLE
dropdowns: fix z-index to be above other content

### PR DESCRIPTION
**Pull request by chelliah**

### CODE REVIEW

File: src/components/DropDownMenu.js [hunk 1]
```diff
@@ -73,7 +73,8 @@ export default class DropDownMenu extends React.Component {
 		        opacity: 0,
 		        transition: 'opacity 0.15s ease-out, margin-top 0.2s ease-out',
 		        pointerEvents: 'none',
-		        color: '#666'
+                color: '#666',
```
Comment (GuruM) [inline src/components/DropDownMenu.js]: Indent

Comment (chelliah) [inline src/components/DropDownMenu.js]: the linting file looks like it got fucked up, might just go ahead and fix that on a separate pr

Comment (chelliah) [inline src/components/DropDownMenu.js]: Unless I fix the whole file by hand there are gonna be indentation issues

**Comment by GuruM** [comment]
👍
